README.md

```diff
@@ -65,4 +65,14 @@ The key and token are configured in the .env file in this project:
 - TRELLO_URL=https://www.trello.com
 - TRELLO_KEY={your key}
 - TRELLO_TOKEN={your token}
-- TRELLO_BOARD={your board}
\ No newline at end of file
+- TRELLO_BOARD={your board}
+
+Added ancible inventory and playbook with the following tasks
+- commit git files
+- install poetry
+- copy poetry into todoapp
+- template added copy env values and prompt for trello token
+- todoapp service created so todoapp can be started automatically
+- todoapp service task to copy it to correct directory
+- task to start poetry and todo app
```

Comment:
Rather than documenting what steps the future person needs to solve, we should document exactly how they should solve it - which really means we should check the `playbook.yaml` and `inventory` files into this codebase! We've already worked to solve this problem, we don't want the next person to have to start again from scratch (and even more so if the next person might also be us!)